internal/adapters/forwarder/events_forwarder/events_forwarder.go

```diff
@@ -91,7 +91,7 @@ func (f *eventsForwarder) ForwardPlayerEvent(ctx context.Context, eventAttribute
 			RoomId: eventAttributes.RoomId,
 		},
 		EventType: fromPlayerEventTypeToGrpcPlayerEventType(eventAttributes.EventType),
-		Metadata:  f.mergeInfos(forwarder.Options.Metadata, eventAttributes.Other),
+		Metadata:  f.mergePlayerInfos(eventAttributes.Other, forwarder.Options.Metadata),
```

Comment:
The logic was wrong here, had to fix it: v9 (plugins/grpc/forwarder.go:392, mergePlayerInfos method)